src/deploy-config.sh

```diff
@@ -133,21 +133,21 @@ echo "The base64 encoded NGINX configuration tarball"
 echo "$encoded_config_tarball"
 echo ""
 
-# Synchronize the NGINX configuration tarball to the NGINX for Azure deployment.
+# Synchronize the NGINX configuration tarball to the NGINXaaS for Azure deployment.
 
 uuid="$(cat /proc/sys/kernel/random/uuid)"
 template_file="template-$uuid.json"
 template_deployment_name="${nginx_deployment_name:0:20}-$uuid"
 
-wget -O "$template_file" https://raw.githubusercontent.com/nginxinc/nginx-for-azure-deploy-action/487d1394d6115d4f42ece6200cbd20859595557d/src/nginx-for-azure-configuration-template.json
+wget -O "$template_file" https://raw.githubusercontent.com/nginxinc/nginxaas-for-azure-deploy-action/487d1394d6115d4f42ece6200cbd20859595557d/src/nginxaas-for-azure-configuration-template.json
```

Comment:
We should avoid renaming the repo or any references to it for now. We can take that on separately if the current repo name becomes a source of confusion for users.